test/powershell/engine/Help/HelpSystem.Tests.ps1

```diff
@@ -201,3 +201,25 @@ Describe "Validate about_help.txt under culture specific folder works" -Tags @('
         $help | Should BeExactly "Hello"
     }
 }
+
+Describe "Get-Help should find help info within help files" -Tags @('CI', 'RequireAdminOnWindows') {
+    It "Get-Help should find help files under pshome" {
+        $helpFile = "about_testCase.help.txt"
+        $culture = (Get-Culture).Name
+        if (!(Test-Path $PSHOME\en-US))
```

Comment:
Also, does this tests run on Linux? If it does, then please use Join-Path instead of "$path\value"
E.g.,
$helpContentPath = join-path  $PSHOME $culture
if (-not (test-path $helpContentPath))
{
}